common/djangoapps/student/tests/test_events.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""
+Test that various events are fired for models in the student app.
+"""
+from django.test import TestCase
+
+from student.tests.factories import UserFactory
+from student.tests.tests import UserProfileEventTestMixin
+
+
+class TestUserProfileEvents(UserProfileEventTestMixin, TestCase):
+    """
+    Test that we emit field change events when UserProfile models are changed.
+    """
+    def setUp(self):
+        super(TestUserProfileEvents, self).setUp()
+        self.user = UserFactory.create()
+        self.profile = self.user.profile
+
+    def test_change_one_field(self):
+        """
+        Verify that we emit an event when a single field changes on the user
+        profile.
+        """
+        self.profile.year_of_birth = 1900
+        self.profile.save()
+        self.assert_profile_event_emitted(
+            year_of_birth={'old_value': u'None', 'new_value': unicode(self.profile.year_of_birth)}
```

Comment:
@dan-f I'm surprised that this is getting evented as a string instead of an integer. Why was that decision made?